service/controller/resource/monitoring/prometheus/resource.go

```diff
@@ -246,9 +246,14 @@ func toPrometheus(ctx context.Context, v interface{}, config Config) (metav1.Obj
 			EvaluationInterval: promv1.Duration(config.EvaluationInterval),
 			Retention:          promv1.Duration(config.RetentionDuration),
 			RetentionSize:      promv1.ByteSize(config.RetentionSize),
+			// Fetches Prometheus rules from any namespace on the Management Cluster
+			// using https://v1-22.docs.kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-metadata-name
 			RuleNamespaceSelector: &metav1.LabelSelector{
-				MatchLabels: map[string]string{
-					"name": key.NamespaceMonitoring(),
```

Comment:
We're removing this constraint, but do we really want to fetch ALL rules?
I mean, it's on MCs so chances are probably very low someone deploys some unknown rules, but I thought aligning on servicemonitors, using the "giantswarm.io/team" label, could have been nice?
Not defined yet on our rules though.

Reply:
The issue here is that this is a `rule namespace selector` and we need to set it to something otherwise the prometheus will only have the rules present in it's namespace.

If we wanted to add the `giantswarm.io/team` label on the rules, we would need to change it in the `rule selector` instead.

Considering customers can now deploy things on MCs, maybe we want to reduce the scope to our apps.

That means I will create the PR on prometheusrules first then release PMO

Reply:
The new commit addresses that :) I'll update the prometheus-rules app when I have time :D

Reply:
Testing will happen tomorrow

Reply:
Yes, looking good :smile: